### PR TITLE
Disabling two-factor authentication documentation update

### PR DESCRIPTION
**Pull request by georgethemarmot**
### Why:
Documentation was not up to date with website changes

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Image and description of where to disable 2fa

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@georgethemarmot Thank you for opening a PR! ✨ While your changes appear to be correct, I'll get this triaged for review by our technical writing team to validate content / style changes 💛

**Comment by kashyapanjali** [comment]
how can i contribute ?

**Comment by nguyenalex836** [comment]
@kashyapanjali Hello! 👋 While there aren't any opportunities to contribute regarding this PR, take a look at our [help wanted section](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) to find open issues you can work on ✨

File: content/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account.md
Comment (yyy7u) [inline content/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account.md:43]: georgethemarmot:main

**Review by subatoi — CHANGES_REQUESTED** [review]
Hi @georgethemarmot—

Thanks for raising this PR and identifying the discrepancy. You're absolutely right that we need to change this, but I've amended your proposals to fit with our style guide.

One more thing is that the image you've proposed doesn't conform to the guidelines in https://docs.github.com/en/contributing/writing-for-github-docs/creating-screenshots so I created a new one that I'll add in a comment below, that you can add to your branch.

Once you've accepted all the changes we can get this merged. Many thanks!

**Comment by subatoi** [comment]
disable-two-factor-authentication

New image to use

**Comment by nguyenalex836** [comment]
@georgethemarmot Howdy! 👋 We wanted to check in and see if you've had a chance to review @subatoi's suggestions, including the new image he provided! The branch conflicts will also need to be resolved as well 💛 

If you'd rather us fix this internally, we're more than happy to do so!

**Comment by subatoi** [comment]
@georgethemarmot Just to let you know, I'm shipping a fix for this internally and have added you as a co-committer (since you use a no-reply), so we can leave this closed.

Thanks for spotting this and your interest in the docs!

**Comment by georgethemarmot** [comment]
@nguyenalex836  @subatoi  
Sorry I had no time to reply before. 
Next time I'll maybe just report it instead of opening PR.

**Comment by subatoi** [comment]
> Sorry I had no time to reply before.

No problem at all!

> Next time I'll maybe just report it instead of opening PR.

In general, that's often the best way to do things. We don't mind people raising PRs first, but since we have to conform to our style guide and restrictions around images (like in this case), it can introduce a bit of friction. If users raise an issue first, it gives us chance to help shape the changes, meaning, typically, less friction. Obviously there isn't much point in raising an issue first if you're only fixing a typo, though!

In this case, I went ahead and pushed a fix because you'd correctly identified something that might have been tripping people up, so thank you again!

You can see your co-committer attribution here:

https://github.com/github/docs/commit/1368bab5336e46d95d15cacd43aeb955d1353e54

**Pull request closed without merging**
